File: Requests/Requests.Application/ViewModels/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Requests.Application.ViewModels
{
    public class Product
    {
        public required string ProductId { get; set; }
        public uint Quantity { get; set; }
        public bool IsCustomerPaying { get; set; }
        public required string Description { get; set; }
    }
}

File: Leader_Mobile/SEP490-Mobile-LeaderApp-main/model/product.ts
export interface Product {
    productId: string;
    name: string;
    description: string;
    quantity?: number
    imageUrl: string;
    inOfStock?: number;
    totalPrice?: number,
    warantyMonths: number;
    priceByDate: number;
    isCustomerPaying: boolean
}